x-pack/plugins/alerting/server/rules_client/common/retry_if_bulk_edit_conflicts.test.ts

```diff
@@ -28,6 +29,11 @@ const mockSuccessfulResult = {
     { id: '2', type: 'alert', attributes: { name: 'Test rule 2' }, references: [] },
   ],
   errors: [],
+  skipped: [
+    { id: 'skip-1', name: 'skip-1', skip_reason: 'RULE_NOT_MODIFIED' as BulkEditSkipReason },
+    { id: 'skip-2', name: 'skip-2', skip_reason: 'RULE_NOT_MODIFIED' as BulkEditSkipReason },
+    { id: 'skip-5', name: 'skip-5', skip_reason: 'RULE_NOT_MODIFIED' as BulkEditSkipReason },
```

Comment:
You do not need these castings here.

Reply:
I do actually, otherwise I get this type error:
![image](https://user-images.githubusercontent.com/5354282/208158188-d602de7a-3e9c-4a07-9c6f-0dd515827a33.png)

Reply:
You are right. My linter behaved weird.